.gitlab-ci.yml

```diff
@@ -35,6 +35,18 @@ variables:
   - export VERSION
   - echo "Version = ${VERSION}"
 
+test-macos:
```

Comment:
do we want to keep this?

Reply:
Probably not if we merge the PR (will really slowdown CI of every PR), but I activated it last week because in some case with heapsize failures occurs after build and I wanted to assert it was not the case.
If we only use the PR to patch for the build (not sure if it is possible or even a good idea) we can keep it.

Reply:
I switch it off, for reference https://gitlab.parity.io/parity/parity-ethereum/-/jobs/118672